### DISCUSSION TITLE
Do you have plan for add test-container for event-store?

### DISCUSSION THREAD

**Discussion by meysamhadeli** [testcontainers/testcontainers-dotnet#728] [Q&A] [answered]
Do you have plan for add test-container for event-store?
I didn't find any resource about that in documentation.
Do you have any advice for handling event-store if test-container does not contain a configuration for it?

Thanks for helping :)

**Comment by HofmeisterAn** [comment, marked as answer]
Do you refer to EventStoreDB? I am finalising the first [PR](https://github.com/testcontainers/testcontainers-dotnet/issues/493) that adds a proper module support. This allows anyone to implement and share their own modules in the future. OC, EventStoreDB could be part of TC's modules too. Are you interested in a contribution?

**Reply by meysamhadeli** [reply]
Yes, I can add this module. it's ok. :)
I will add event-store module to test-container.

Thanks.

**Reply by HofmeisterAn** [reply]
It might be a good idea to wait until the PR is done, but you can try to get familiar with the API and prepare the module in the meantime. Unfortunately, I am not done with the documentation yet. If you have any questions you can ping me on Slack. Here is a small [PoC](https://github.com/testcontainers/testcontainers-dotnet/blob/aaaf016d68ed04afb20940805e46e6e02cdbe4a4/src/Testcontainers.Module.Kafka/KafkaBuilder.cs) that follows the most principles. You find the latest changes in feature/prepare-dedicated-modules.

**Reply by meysamhadeli** [reply]
It's ok, I start to develop this feature base on principles that you mentioned and If I have any questions, I will contact you, if I any question.

Thanks.

**Reply by HofmeisterAn** [reply]
Just join https://slack.testcontainers.org/. I am using the same name on both platforms. I might not be very responsive this week. Next week I am definitely available again.

**Reply by meysamhadeli** [reply]
Thanks :)
